src/app/app.component.html

```diff
@@ -1,8 +1,8 @@
-<span class="tasks">
-   <a><img src = "task/task.jpg" alt="" height="20" weight="20"></a>
+<img src="task/task.jpg" alt="xxx" height="20" width="20">
```

Comment:
If you are adding alt don't give it this kind of message. Like I've said it actually has a proper use case.